Readme.md

```diff
@@ -0,0 +1,95 @@
+Rubric for Module 13
+--------------------
+
+The app must have a README file including: 
+
+a) A basic explanation of the app: The app is nearly complete und is working, so I hope, that no explanation is necessary.
+
+b) Explanations of how your app will fulfill the final submission rubric requirements (see next section).
+    
+c) A list of planned screens and elements: There are already complete.
+
+d) The app has a splash screen suitable for the app. It can be either a static or animated splash screen: Is in place, I show a text message with animation built in.
+
+
+
+
+Rubric for week 14
+------------------
+
+Now I discuss the points of the required Final Capstone App Functionality needed to pass in week 14:
+
+a) We suggest using Xcode 14.3 or 14.3.1 and iOS 16.x, rather than Xcode 15 and iOS 17.
+
+Here I use Xcode 15 and iOS 17, since Xcode 15 is installed on my mac mini. I do not even know, if I could use Xcode 14 in an easy way.
```

Comment:
We only suggested the older because the videos show that interface and it changed with Xcode 15. Some students panic. You are fine.